hc/api/models.py

```diff
@@ -57,6 +57,7 @@ class Meta:
     nag_after = models.DateTimeField(null=True,blank=True,editable=False)
     status = models.CharField(max_length=6, choices=STATUSES, default="new")
     often = models.BooleanField(default=False)
+    priority = models.IntegerField(default=0)
```

Comment:
Great implementation of priority with integers. A list of strings could also work.